omero/sysadmins/unix/server-centos6-ius.txt

```diff
@@ -0,0 +1,217 @@
+Installing Python 2.7 and Ice 3.5 on CentOS 6
+=============================================
+
+Running OMERO on CentOS 6 has a number of special requirements which
+deviate from the standard installation instructions.  The instructions
+below will set up Python 2.7 and Ice 3.5 on CentOS 6. We tested the
+installation with Python 2.7 from `IUS <https://ius.io/>`_ and used
+a `virtual environment <https://virtualenv.readthedocs.org/en/latest/>`_. 
+
+Python 2.7
+----------
+
+CentOS 6 provides Python 2.6.  However, OMERO.web requires Python 2.7
+in order to use django 1.8.  While django 1.6 may be used with Python
+2.6, this version of django no longer has security support.  In
+consequence, it is necessary to upgrade to Python 2.7 in order to
+obtain django security updates, which are required for a production
+deployment.
+
+As root, run::
+
+    wget https://centos6.iuscommunity.org/ius-release.rpm
+    rpm -Uvh ius-release*.rpm
+    yum install python27
+
+To use this version of Python, it is preferrable to install
+the various dependencies in a virtual environment. 
+If virtualenv is not already installed, install it using pip run::
+
+	yum install python27-pip
+	pip2.7 install virtualenv
+
+To demonstrate its use::
+
+    $ python --version
+    Python 2.6.6
+    $ virtualenv /home/omero/omeroenv
```

Comment:
Maybe safer to use ``-p`` flag when creating your virtualenv instance to specify the Python executable